packages/picasso.js/src/core/chart-components/labels/strategies/slices.js

```diff
@@ -8,6 +8,20 @@ function pad(bounds, padding) {
   bounds.height -= (padding * 2);
 }
 
+/**
+ * Check if the pie slice label is located entirely within it's parent rect
+ * @param {Object} bounds - An object describing the position(x,y), width and height of label
```

Comment:
both parameters should be of type `rect`

Reply:
so it should be @param {rect} bounds?

Reply:
Yes

Reply:
fixed!